.drone.star

```diff
@@ -1161,11 +1161,15 @@ def uiTestPipeline(ctx, filterTags, runPart = 1, numberOfParts = 1, storage = "o
 def e2eTests(ctx):
     test_suites = {
         "suite1": {
-            "path": "tests/e2e/cucumber/features/*/*[!.oc10].feature",
+            "path": "tests/e2e/cucumber/features/{smoke,journeys}/*.feature",
```

Comment:
I was thinking of a more dynamic approach because the e2e tests are likely to increase. But it's good until we can come up with a dynamic approach.